pyproject.toml

```diff
@@ -56,6 +56,10 @@ dependencies = [
     'rdt>=1.17.0',
     'sdmetrics>=0.20.1',
     'sdv>=1.21.0',
+    'openpyxl>=3.0.0',
+    'kaleido>=0.2.1',
+    'pillow>=9.0.0',
+    'pydrive2>=1.3.1'
```

Comment:
@amontanez24, let me know if adding those dependencies to sdgym makes sense:
- `openpyxl` is for saving the xlsx file as needed
- `kaleido` and `pillow` are for generating and saving the graph in the xlsx file. I thought it might be nice to have a preview of the plot directly, but if not, those can be removed.
- `pydrive2` is for uploading to GDrive